pkg/crc/machine/machine.go

```diff
@@ -378,6 +378,23 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		return *result, errors.Newf("Error starting kubelet: %s", err)
 	}
 	if kubeletStarted {
+		// With 4.3, when cluster come up and request header client-ca regenerated then
+		// openshift-apiserver pod is not able to consume it and as a result it shows
+		// `certificate has expired or is not yet valid` in log message. To resolve
```

Comment:
"openshift-apiserver pod is not able to consume it and as a result it shows `certificate has expired or is not yet valid` in log message. " This part is not accurate, right after boot, this "certificate has expired or is not yet valid" message appears in the logs, and `client-ca` is not set in the config map. Then the certificates and CAs are regenerated, `client-ca` shows up in the config map, and then the message changes to "unknown authority"